src/newSeed/baseStage.scss

```diff
@@ -1,66 +1,100 @@
 @import "styles/colors";
 // This styles are common to all the stages components
 // Theme for the stages
-@mixin stageTheme {
-  * {
-    font-size: 1rem;
+.infoContainer {
+  font-size: 1rem;
+  font-family: Inter;
+  background-color: $color-black-3;
+  border: 1px solid $color-darkpink-2;
+  border-radius: 1.5em;
+  padding: 2em;
+  min-width: 40rem;
+  .title {
+    font-size: 1.625em;
+    color: $color-white;
+    line-height: 2em;
+    font-weight: bold;
   }
-  .infoContainer {
-    background-color: $color-black-3;
-    border: 1px solid $color-darkpink-2;
-    border-radius: 1.5em;
-    padding: 2em;
-    min-width: 40rem;
-    .title {
-      font-size: 1.625em;
-      color: $color-white;
-      line-height: 2em;
-      font-weight: bold;
-    }
-    .subtext {
-      line-height: 1.5em;
-      font-size: 1em;
-      color: #dcdae5;
-      margin-bottom: 5em;
-    }
-    .infoInput {
-      margin-bottom: 2em;
-    }
-    .buttonContainer {
-      display: flex;
-      flex-flow: row wrap;
-      align-items: center;
-      column-gap: 3em;
-      font-weight: bold;
-      margin-top: 5em;
-
-      .cancel {
-        color: $color-orange;
-        font-family: Inter;
-      }
-    }
-    label {
-      display: block;
-      font-size: 1.1em;
-      line-height: 2em;
-      color: $color-white;
-    }
-    input {
-      width: 26em;
-      height: 2.7em;
-      border: 1px solid $color-darkpink-2;
+  .subtext {
+    line-height: 1.5em;
+    font-size: 1em;
+    color: #dcdae5;
+    margin-bottom: 5em;
+  }
+  .infoInput {
+    margin-bottom: 2em;
+  }
+  .buttonContainer {
+    display: flex;
+    flex-flow: row wrap;
+    column-gap: 3em;
+    font-weight: bold;
+    margin-top: 5em;
+    .cancel {
+      cursor: pointer;
+      color: $color-orange;
+      font-family: Inter;
     }
-    a {
-      color: #989798;
-      text-decoration: underline;
+  }
+  .stageButton {
+    display: flex;
+    align-items: center;
+  }
+  label {
+    display: block;
+    font-size: 1.1em;
+    line-height: 2em;
+    color: $color-white;
+  }
+  input {
+    width: 26em;
+    height: 2.7em;
+    background-color: #1b1722;
+    border: 1px solid $color-darkpink-2;
+    border-radius: 0.6em;
```

Comment:
should be 10px according to what i see in the mock.  I think we should use 10px.